tests/test_virtualenv.py

```diff
@@ -230,3 +230,26 @@ def test_trims_path(self, virtual_env):
             '/0ho8Ke/hL9DsW/ymH81W:/L51pua:/Vngp3V/G2m7Ih/05m7qW/LyrYkK/'
             'l5NuwA/oq1DPp:/Zpdhu4/bjvuqt:/STGtcb/FhAnWH/HwTvOr/gngGiB:'
             '/Zizj4D/szncsv/O5wO6X/joFHVT')
+
+
+class TestVirtualEnvProperties(object):
+    @patch("plpacker.virtualenv.FileSet")
+    @patch.object(VirtualEnv, "site_package_dirs", new_callable=PropertyMock)
```

Comment:
@Naxest,
You have to import `PropertyMock` or else this will not work.

```
try:
    from unittest.mock import patch, sentinel, PropertyMock
except ImportError:
    from mock import patch, sentinel, PropertyMock
```

Did this actually run when you ran `pytest`?

Reply:
Yes, sorry, I forgot to commit that imports.